xml/rmt_install.xml

```diff
@@ -31,7 +33,232 @@
    possible for a single server to host both simultaneously.
   </para>
  </warning>
+ <sect1 xml:id="sec-rmt-storage-requirements">
+     <title>Storage Requirements</title>
+     <para>
+         Downloaded packages are stored in <filename>/var/lib/rmt/public/repo/SUSE/Products/</filename>. 
+         The amount of storage your &rmt; server requires is dependent on several variables: the 
+         number of  repositories and architectures that you mirror, how many packages are 
+         downloaded, the  number of  extensions that are enabled, and the age of your server installation. 
+         Outdated  packages are not removed by &rmt;, but accumulate over time. When new versions are 
+         downloaded, the old versions are kept on disk but are not referenced by repository 
+         metadata, which makes them invisible to clients.
+     </para>
+     <para>
+         As a general guide, two to three times the total size of all mirrored repositories should be 
+         sufficient. If you are mirroring a single release, such as &slea; 15.1, 250 GB provides room
```

Comment:
`a single release` -- that seems confusing. It's not just SLES 15.1, it's SLES15.1 + all extensions (incl. PackageHub).

Also the figures seem rather arbitrary -- in principle there shouldn't be much difference between 15.0 and 15.1, in fact 15.1 repos are smaller. And yet the suggested figure for 15.1 is 120 GB higher than for 15.0.

`As a guidline, 200 GB is sufficient to mirror all repos for any SLES release with all extensions`?